pkg/build/apis/build/types.go

```diff
@@ -463,7 +463,7 @@ const (
 
 	// StatusReasonExceededRetryTimeout is an error condition when the build has
 	// not completed and retrying the build times out.
-	StatusReasonExceededRetryTimeout StatusReason = "ExceededRetryTimeout"
+	StatusReasonExceededRetryTimeout StatusReason = "ExceededRetryTimeout" //TODO not used currently, api or v1
```

Comment:
not sure what the todo(action) would be?  probably worthy of note but not a todo (we can't remove them since they are apis and there might still be builds out there somewhere that have these values)

Reply:
comment will be switched to a simple note/fyi